### PR TITLE
Multi-shop options for add / edit actions

### PR DESCRIPTION
**Pull request by qtheuret**
I worked on multi-shops environment and Prestashop 1.7.5 version.

I made some modifications on the prestapyt library to manage multi-shop options for add / edit actions of the Prestashop API.

The PHP Library for Prestashop has been updated with the `id_group_shop` option : https://github.com/PrestaShop/PrestaShop-webservice-lib/blob/master/PSWebServiceLibrary.php#L224

### CODE REVIEW

File: prestapyt/prestapyt.py [hunk 1]
```diff
@@ -294,7 +294,7 @@ def _validate_query_options(self, options):
             )
         supported = (
             'filter', 'display', 'sort',
-            'limit', 'schema', 'date', 'id_shop'
+            'limit', 'schema', 'date', 'id_shop', 'id_group_shop',
```
Comment (pedrobaeza) [inline prestapyt/prestapyt.py]: Isn't `id_shop` already for multi-shop?

Comment (qtheuret) [reply prestapyt/prestapyt.py]: I'm agree with this, but this requires that you create one call for each shop, even if the shops are in the same group.
The `id_group_shop` option allows you to update data for each shops in the same group with one call.

Comment (pedrobaeza) [reply prestapyt/prestapyt.py]: I don't figure out right now how to update multiple data from different shops...

Comment (qtheuret) [reply prestapyt/prestapyt.py]: It's not for requests Prestashop -> Odoo but for requests Odoo -> Prestashop such as product updates with `connector_prestashop_catalog_manager` Odoo module.

It's simply an update to match with the PHP Library : https://github.com/PrestaShop/PrestaShop-webservice-lib/blob/master/PSWebServiceLibrary.php#L341

**Comment by pedrobaeza** [comment]
@FFernandez-PlanetaTIC what do you think about this?

**Comment by FFernandez-PlanetaTIC** [comment]
LGTM, @qtheuret thanks!

**Comment by pedrobaeza** [comment]
Ok, let's merge it.

**Pull request merged**